### ISSUE TITLE
[Creature] Gravemaster Golems

### ISSUE THREAD

**Issue by AquaticAnimal** [AlexModGuy/Ice-and-Fire-Suggestions#1310]
The Gravemaster Golem are just stone statues with a hammer infront of the Mausoleum until you open it with a key then the golems break from stone form into a basically a iron golem but with a gravemaster sort of style to it.

Drops: Gravemasters war-hammer (3.7 drop rate), Dread shards (guaranteed 3-5) and silver ingots (1-2)

**Comment by CTH999** [comment]
yeah! and, perhaps they can somehow request "backup" from Lich's? That could be an interesting mechanic, to give more uses to the Lich's.

Or, maybe Lich's can _also_ summon these (if within a certain range of the Mausuleum). That gives you a reason to try to avoid Lich's while opening this, cause the more of these they summon, the more you will have to attack!
